Documentation/ceph-cluster-crd.md

```diff
@@ -301,7 +303,7 @@ Configure the network that will be enabled for the cluster and services.
 
 To use host networking, set `provider: host`.
 
-#### Multus (EXPERIMENTAL)
+#### Multus
 
 Rook has experimental support for Multus.
 Currently there is an [open issue](https://github.com/ceph/ceph-csi/issues/1323) in ceph-csi which explains the csi-rbdPlugin issue while using multus network.
```

Comment:
We were waiting for a new ceph-csi release to take multus out of experimental right?

Reply:
Which one?

Reply:
https://github.com/ceph/ceph-csi/issues/1323 is closed but the fix is not in ceph-csi release v3.2.1 and is in master.
So we were waiting for a new ceph-csi release right?

Reply:
cc @Madhu-1